### PR TITLE
changelog: prepare for 0.0.2

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,25 @@
...
 
+## [0.0.2] - 2018-02-21
+
+### Added
+* Add `standard` for linting but only print warnings for now (@feross)
+* Add Open Open Source `CONTRIBUTING.md` (@ralphtheninja)
+
+### Changed
+* Update [dependencies](https://github.com/airtap/airtap/pull/41) (@ralphtheninja)
+* Renamed `bin/airtap` to `bin/airtap.js` + fix standard (@ralphtheninja)
```
Comment (vweevers) [inline CHANGELOG.md]: Nit: renamed => rename

**Comment by vweevers** [comment]
@ralphtheninja return the favor? https://github.com/Level/abstract-leveldown/pull/208 😀

**Comment by ralphtheninja** [comment]
> @ralphtheninja return the favor? Level/abstract-leveldown#208 grinning

Haha, I just noticed we basically made them at the same time :smile:

**Comment by ralphtheninja** [comment]
Will squash in a bit.

**Pull request merged**